edns.go

```diff
@@ -658,56 +660,40 @@ func (e *EDNS0_LOCAL) unpack(b []byte) error {
 // the TCP connection alive. See RFC 7828.
 type EDNS0_TCP_KEEPALIVE struct {
 	Code    uint16 // Always EDNSTCPKEEPALIVE
-	Length  uint16 // the value 0 if the TIMEOUT is omitted, the value 2 if it is present;
```

Comment:
We can't removed an exported field on an exported struct. That's a no go.

Reply:
No valid keepalive edns could be parsed or generated with the current code, so it is unlikely somebody reference this field that should not exist in the first place.

Reply:
@tmthrgd can we keep it but mark deprecated and just not use it?

Reply:
I don't necessarily object to that. I'm happy to review a change that does that and see what it's like.

Reply:
I don't mind either way, breaking users is a compile time error (good! Not some fuzzy thing failing at runtime) and a fairly easy fix (remove setting the field). OTOH, keeping it (for now) and marking it unused in the docs doesn't really cost us anything

Reply:
Length is back, marked deprecated now.